torch_xla/csrc/tensor.cpp

```diff
@@ -639,8 +639,9 @@ c10::SymNode XLASymNodeImpl::add(const c10::SymNode& other) {
 }
 
 c10::SymNode XLASymNodeImpl::sub(const c10::SymNode& other) {
-  XLA_CHECK(false) << "XLASymNodeImpl::" << __FUNCTION__
```

Comment:
Wonder if we should have something like:
```
TORCH_LAZY_FN_COUNTER("xla::size_");
```

Reply:
I feel it's ok not to increase the counter here because it's upstream that redirect the "-" operator to the `sub` method. Also, without the change here, the test will fail with error `XLASymNodeImpl::sub not implemented`

Reply:
The TORCH_LAZY_FN_COUNTER is more or less a way to suggest that our version of the op is called instead of the fallback one. The MARCO actually will include the method name in the counter: xla::size_sub.

Reply:
done.